composer.json

```diff
@@ -31,6 +31,7 @@
         "neos/composer-plugin": "^2.0",
         "composer/composer": "^1.9 || ^2.0",
         "egulias/email-validator": "^2.1",
+        "typo3fluid/fluid": "~2.5.11 || ^2.6.10",
```

Comment:
Removing that will make tests fail, because fluid will indeed not get installed. Fluid-Adaptor is in the collection and auto-exists, and hence all tests for that are also run, but as it's not composer required, it's own dependencies will not be installed.